fcrepo-http-api/src/main/java/org/fcrepo/http/api/ExternalContentPathValidator.java

```diff
@@ -117,12 +117,20 @@ public void validate(final String extPath) throws ExternalMessageBodyException {
         throw new ExternalMessageBodyException("Path did not match any allowed external content paths: " + extPath);
     }
 
-    private String normalizePath(final String path) {
+    private String normalizeUri(final String path) {
+        // lowercase the scheme since it is case insensitive
+        final String[] parts = path.split(":", 2);
+        final String normalized;
+        if (parts.length > 1) {
```

Comment:
It seems that you are only handling the case of `parts.length == 2`. Should the check be looking for a length of 2?